pallets/stellar-relay/src/traits.rs

```diff
@@ -1,7 +1,7 @@
 use codec::{Decode, Encode, MaxEncodedLen};
-use frame_support::{traits::Get, BoundedVec, RuntimeDebug};
+use frame_support::{traits::Get, BoundedVec, pallet_prelude::RuntimeDebug};
 use scale_info::TypeInfo;
-use serde::{Deserialize, Serialize};
+use sp_runtime::serde::{Deserialize, Serialize};
```

Comment:
Although it's probably the same `serde`, we better use `serde` directly to avoid confusion like in all other pallets.

By the way, It took me a while to change this because I was getting errors by just changing this import, and turns out that a `cargo clean` had to be done.. Very strange.